### DISCUSSION TITLE
AI tagging not activating/working

### DISCUSSION THREAD

**Discussion by GraceHough** [elblogbruno/NotionAI-MyMind#18] [Q&A]
Hello! I went through all the server setup and My Mind Structure instructions, however, the TensorFlow ai tagging doesn't seem to be doing anything at all. Is there another step I'm missing or do I need to be training the ai myself? I'm very new to ai image recognition so I wouldn't know if this is obvious. I've double-checked my settings for the ai tag property title and made sure they matched in notion and on the server.
Thank you.

**Comment by elblogbruno** [comment]
Oh I am sorry for it!
I will try to help you out!
If you add any content does it tell you for example like here, "no-tags-available"? 
![image](https://user-images.githubusercontent.com/10481058/116722255-75da3280-a9de-11eb-8ac9-cc5d111672f0.png)

Also when you add content, does it print something strange on the console? If you want you can share the log with me by accessing your server IP and adding /log. For example http://server-ip:port/log. Be careful to remove the token V2 from the log if you share it here, we don't want people to enter your Notion account.

Hope these hints help you out!
